libraries/botbuilder-dialogs-adaptive-testing/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "botbuilder-dialogs-adaptive-testing",
+  "private": true,
```

Comment:
This package will eventually be shipped, and should possibly be going out as a preview package. @luhan2017, @chon219 is this package ready for preview consumption?

Reply:
There's some background talk that this library might be a composer only tool, let's not block on this for now (this library has not been published to npm or MyGet yet).